templates/includes/install-problems.hbs

```diff
@@ -2,28 +2,42 @@
+    This is a known behavior when the battery charge is very low. The watch usually recovers from this condition by simply boot cycling for some time. All watches also charge in fastboot or recovery mode. Letting the watch dwell on the fastboot or recovery menu screen for some time should fix the problem even faster.
 </div>
-
+<h2>Flashing problems</h2>
 <div class="callout">
-    <h4>Flash was successful with fastboot throwing no errors, but watch does not boot</h4>
-    The flashing process can silently fail due to an unstable connection. Make sure you cleaned the contacts on the cradle and the watch. Redo the flashing process up to four times. On watches with especially bad connection, it can help to fixated the watch to the cradle using ductape or rubber bands.
-</div>
-
-<div class="callout">
-    <h4>My fastboot command displays "invalid sparse file format at header magic" when starting to flash the device</h4>
-    This is not a fatal error and can safely be ignored.
-</div>
-
-<div class="callout">
-    <h4>My fastboot command crashes or hangs at "invalid sparse file format at header magi" (with a missing "c" in magic)</h4>
-    This error occurs when using deprecated fastboot and ADB commands on Windows systems. If you already had those commands installed and skipped downloading our <a href="https://release.asteroidos.org/tools/adb_1_0_39+fastboot+ext2simg.zip">supplied zip file</a>, please install those commands instead.
-</div>
-
-<div class="callout">
-    <h4>My Asus Zenwatch 2 flashed successfully but does not boot</h4>
-    There are two versions of the Asus Zenwatch 2, codenamed Sparrow and Wren. Their images are not interchangeable. Please ensure you are using the right files.
+    <h3>Watch is not detected and all fastboot commands fail</h3>
+    <ul>
+      <li><h4>When using Linux,</h4> try to prepend <code>sudo</code> to the commands to execute them with root rights. If that works, you are missing UDEV rules for Android devices that would allow your user to issue the commands. Many package managers offer the <code>android-udev-rules</code> package to make the manual entry of those udev rules unnecessary.</li>
+      <li><h4>On Windows systems,</h4> this usually happens due to driver problems. If you have any other Android USB driver installed than the official Google USB driver we link above in the install boxes, please remove that one. Then do a clean install of the Google USB driver or update it by installing a possibly newer version.<br>If that does not solve your problem, we highly recommend to not waste time trying to debug the issue on windows. But create a persistent live Linux USB-Stick according to <a href="https://askubuntu.com/questions/1300454/easy-full-install-usb-that-boots-both-bios-and-uefi">e.g. these instructions at askubuntu.com</a>. And then follow the Linux commands above while booted into the live Linux USB-Stick.</li>
```

Comment:
`we highly recommend to not waste time` might be a bit aggressive? I think something like `We find that the linux fastboot drivers are more reliable. It is very easy to create a bootable USB....`

` ...windows. But create ...`
this isn't entirely grammatically correct. ` we highly recommend to not waste time ... on windows, but to create a bootable Linux USB stick...` should be more correct. similarly with `... askubuntu. And then` - this should be `follow these instructions at askubuntu, and then to follow the Linux install commands`